tools/update_tests.sh

```diff
@@ -52,6 +50,11 @@ function update () {
     cd $OLD_DIR
 }
 
+if [[ $1 = "--all" ]]; then
```

Comment:
Very basic option handling.

Reply:
In #471 I've used the `test` keyword instead of double brackets (for portability concerns), maybe we could use it here as well (but not blocking if you choose not to).

Reply:
The script starts with `#!/usr/bin/env bash`. Also I use arrays. I'm not sure it is usefull but I can try to make it more "portable".